Photo Collection/Photo Collection/PhotosCollectionViewController.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  PhotosCollectionViewController.swift
+//  Photo Collection
+//
+//  Created by Cameron Dunn on 1/16/19.
+//  Copyright © 2019 Cameron Dunn. All rights reserved.
+//
+
+import UIKit
+
+private let reuseIdentifier = "Cell"
+
+class PhotosCollectionViewController: UICollectionViewController {
+    let photoController = PhotoController()
+    let themeHelper = ThemeHelper()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Uncomment the following line to preserve selection between presentations
+        // self.clearsSelectionOnViewWillAppear = false
+
+        // Register cell classes
+        self.collectionView!.register(UICollectionViewCell.self, forCellWithReuseIdentifier: reuseIdentifier)
+        // Do any additional setup after loading the view.
+    }
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(true)
+        setTheme()
+        collectionView.reloadData()
+    }
+    func setTheme(){
+        guard let theme = themeHelper.themePreference else {return}
+        if(theme == .dark){
+            collectionView.backgroundColor = UIColor.darkGray
+        }else if(theme == .light){
+            collectionView.backgroundColor = UIColor.white
+        }
+    }
+    override func numberOfSections(in collectionView: UICollectionView) -> Int {
+        // #warning Incomplete implementation, return the number of sections
+        return photoController.photos.count
```

Comment:
This return value is causing the issue. You don't need multiple section
```suggestion
        return 1
```